docs/pages/components/button.md

```diff
@@ -8,116 +8,67 @@ permalink: components/button.html
 folder: components
 ---
 
-Buttons allow users to perform actions.
-{: .docs-intro}
-The priority of buttons within a page should be considered. For instance, only use the main button once within a page or dialog.
-Color is also important. For instance, the most important button has a blue background where as a red button should only be used if the action it performs is potentially destructive.
-
-<br>
-
-## Button Options
-There are three emphasis styles used to indicate the importance of the button on the page.
-- **Emphasized Button**: There should only be one highlighted button on the page. This is the primary call to action.
-- **Regular Button**: The default button style and the most common button. There may be more than one on a page.
-- **Light Button**: This is the lowest priority button and most often used with page content like appearing in a table or list. There may be more than one on the page.
+Buttons allow users to perform actions. All the buttons require base class `fd-button` and additional modifier for all the flavours.
```

Comment:
You need {: .docs-intro} before the text

Reply:
done